mkdocs.yml

```diff
@@ -79,6 +79,8 @@ nav:
     - Extra Types:
       - Color Types: usage/types/extra_types/color_types.md
       - Payment Card Numbers: usage/types/extra_types/payment_cards.md
+      - Phone Numbers: usage/types/extra_types/phone_numbers.md
+      - Routing Number: usage/types/extra_types/routing_number.md
```

Comment:
`Routing Number` or `Routing Numbers`? I can see `Routing Numbers` in `types.md`.

Also, the file name has to be fixed I think. because we have `phone_numbers.md` for `Phone Numbers` but the file name for `Routing Number` is singular.

I would suggest to make all of them plural

Reply:
done